README.md

```diff
@@ -1,47 +1,36 @@
 # greenboot
-Generic Health Check Framework for systemd on [rpm-ostree](https://coreos.github.io/rpm-ostree/) based systems
+Generic Health Check Framework for systemd on [rpm-ostree](https://coreos.github.io/rpm-ostree/) based systems.
+
+## Table of contents
+* [Installation](#installation)
+* [Usage](#usage)
+  + [Health checks with bash scripts](#health-checks-with-bash-scripts)
+    - [Health checks included with subpackage greenboot-default-health-checks](#health-checks-included-with-subpackage-greenboot\-default\-health\-checks)
+  + [Health Checks with systemd services](#health-checks-with-systemd-services)
+    - [Required Checks](#required-checks)
+    - [Wanted Checks](#wanted-checks)
+  + [Configuration](#configuration)
+* [How does it work](#how-does-it-work)
 
 ## Installation
 Greenboot is comprised of two packages:
 - `greenboot` itself, with all core functionalities: check provided scripts, reboot if these checks don't pass, rollback to previous deployment if rebooting hasn't solved the problem, etc.
 - `greenboot-default-health-checks`, a series of optional and curated health checks provided by Greenboot maintainers.
 
-In order to get a full Greenboot installationon Fedora Silverblue, Fedora IoT or Fedora CoreOS:
+In order to get a full Greenboot installation on Fedora Silverblue, Fedora IoT or Fedora CoreOS:
 ```
 rpm-ostree install greenboot greenboot-default-health-checks
 
 systemctl reboot
 ```
 
-## How does it work?
```

Comment:
I think a how it works section makes sense, but maybe this was out of date?

Reply:
Mmm maybe incomplete because it doesn't tell how the status is outputed to the MOTD in case the checks aren't passing, but I would say the rest of the info is up to date.

Reply:
Updated the README with how the status is outputed to the MOTD in case the checks aren't passing